helper/schema/runtime.go

```diff
@@ -0,0 +1,31 @@
+// Copied from https://github.com/pulumi/pulumi-terraform-bridge/blob/7c411fc807e709a85170cab0add5cb3a856c0c25/pkg/tfbridge/runtime.go#L1
```

Comment:
This is a really gnarly approach and I'm a little sad it's getting copied :) This for example does not detect unit test context properly. We don't have context.Context to propagate an option with but perhaps we could use a public global var that can be set by the unit test and TFGEN to enable the check.